ompi/mca/hook/comm_method/hook_comm_method_fns.c

```diff
@@ -84,15 +91,20 @@ lookup_btl_name_for_send(ompi_communicator_t* comm, int rank) {
 // that the caller has to free.
 static char *
 comm_method_string(MPI_Comm comm, int rank, int *comm_mode) {
-    char *p;
+    char *p, *btl;
     char *string = malloc(COMM_METHOD_STRING_SIZE);
 
     if (!string) { return NULL; }
 
     p = lookup_pml_name();
     if (p && 0==strncmp("ob1", p, 4)) {      // BTL
         if (comm_mode) { *comm_mode = MODE_IS_BTL; }
-        mystrncpy(string, lookup_btl_name_for_send(comm, rank), COMM_METHOD_STRING_SIZE);
+        btl = lookup_btl_name_for_send(comm, rank);
+        if (NULL == btl) {
+            mystrncpy(string, "n/a", COMM_METHOD_STRING_SIZE);
```

Comment:
We don't need to do this here, but I would like to remove that `mystrcpy` function.

Reply:
Yes, please delete the 1-line static non-inline function from this file!

Reply:
Done. I squashed the removal of `mystrncpy` onto this commit. Please re-review. Thanks!